testsuite/compat/src/test/java/org/jboss/as/test/compat/jpa/hibernate/transformer/StateType.java

```diff
@@ -58,6 +58,10 @@ private static void internalSessionImplementorUsingMethod(SessionImplementor ses
     }
 
     public Object nullSafeGet(ResultSet rs, String[] names, SessionImplementor session, Object owner) throws HibernateException, SQLException {
+        return internalNullSafeGet(rs, names, session, owner);
+    }
+
+    private Object internalNullSafeGet(ResultSet rs, String[] names, SessionImplementor session, Object owner) throws HibernateException, SQLException {
```

Comment:
Is there a reason for this change?

Reply:
This additional test change recreates a problem reported to us that we wanted to fix.